crates/bevy_pbr/src/render/pbr_functions.wgsl

```diff
@@ -229,6 +242,25 @@ fn pbr(
         }
         let light_contrib = lighting::point_light(in.world_position.xyz, light_id, roughness, NdotV, in.N, in.V, R, F0, f_ab, diffuse_color);
         direct_light += light_contrib * shadow;
+
+        if diffuse_transmission > 0.0 {
```

Comment:
I feel like this may be better as a shader def, rather than a branch? Diffuse transmission seems relativity uncommon (leaves, wax) and we don't want to affect performance for the more common opaque path.

Reply:
Hmm. Yeah, agree. This codepath is probably uncommon enough to make it worth it.

We're close to using up all of the bits of the current `u32` key though, are we okay with making that a `u64`?